scripts/actions/mobskills/aqua_breath.lua

```diff
@@ -14,7 +14,8 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local dmgmod = xi.mobskills.mobBreathMove(mob, target, skill, 0.25, 1.5, xi.element.WATER, 400)
+    -- TODO: Needs a flat bonus 100 damage added to breath
```

Comment:
return dmg + 100

Reply:
Made an adjustment for that if you want to see if that looks okay to you